src/ResultSet.h

```diff
@@ -0,0 +1,17 @@
+#ifndef RESULT_SET_H
+#define RESULT_SET_H
+
+#include <vector>
+
+#include "FontDescriptor.h"
+
+class ResultSet : public std::vector<FontDescriptor *> {
+public:
+  ~ResultSet() {
+    for (FontDescriptor *fd : *this) {
+      delete fd;
+    }
+  }
```

Comment:
This is a footgun waiting to happen, but I realize we're just implementing previous behavior. Can we at least delete copy/move constructor/= operators and leave a comment above the class that this is an owning container (since anything inside it will be deleted when it is)?